include/El/core/DistGraph.hpp

```diff
@@ -11,6 +11,206 @@
 #ifndef EL_CORE_DISTGRAPH_HPP
 #define EL_CORE_DISTGRAPH_HPP
 
-#include <El/core/DistGraph/decl.hpp>
```

Comment:
Is there a reason for purposefully breaking the convention of splitting the declarations and implementations into `decl.hpp` and `impl.hpp` files?